src/IssueViz.Security/HotspotsList/TableDataSource/HotspotTableEntry.cs

```diff
@@ -0,0 +1,128 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Windows;
+using Microsoft.VisualStudio.Shell.TableControl;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+using ImageMoniker = Microsoft.VisualStudio.Imaging.Interop.ImageMoniker;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource
+{
+    internal class HotspotTableEntry : IWpfTableEntry
+    {
+        private readonly IAnalysisIssueVisualization hotspot;
+
+        public HotspotTableEntry(IAnalysisIssueVisualization hotspot)
+        {
+            this.hotspot = hotspot;
+        }
+
+        public object Identity => hotspot;
+
+        public bool TryGetValue(string keyName, out object content)
+        {
+            var originalIssue = hotspot.Issue;
+
+            switch (keyName)
```

Comment:
Presumably hotspots have a severity too?

Reply:
Perhaps we should check with the PM(s) which columns they want to see in the list.

Reply:
No idea, do we get that in the API?
I think Alex said that the priority (high/medium/low) is taken from a spreadsheet, so I'm not sure we get that back. I'll ask Damien,